### PR TITLE
feat: add artifact hub annotations

### PR DESCRIPTION
**Pull request by ksimon1**
**What this PR does / why we need it**:
feat: add artifact hub annotations

this commit adds special artifact hub annotations, which add information about maintainers, recommendations, provider, ... These annotation are used only by artifact hub.
Signed-off-by: Karel Simon <ksimon@redhat.com>

**Release note**:
```
Add artifact hub annotations
```

### CODE REVIEW

File: manifests/kubernetes/kubevirt-tekton-tasks-kubernetes.yaml [hunk 1]
```diff
@@ -8,6 +8,13 @@ metadata:
     tekton.dev/tags: kubevirt
     tekton.dev/displayName: "KubeVirt cleanup VM"
     tekton.dev/platforms: "linux/amd64"
+    artifacthub.io/maintainers: |
+      - name: Karel Simon
+        email: ksimon@redhat.com
+    artifacthub.io/provider: KubeVirt
+    artifacthub.io/recommendations: |
```
Comment (0xFelix) [inline manifests/kubernetes/kubevirt-tekton-tasks-kubernetes.yaml:15]: Makes sense to put the repo URL here too?

Comment (codingben) [reply manifests/kubernetes/kubevirt-tekton-tasks-kubernetes.yaml:15]: Hi, there is already repo link in the ArtifactHub repository, e.g. [1] you can see source link.

[1] https://artifacthub.io/packages/tekton-task/kubevirt-disk-uploader/kubevirt-disk-uploader-task

**Review by 0xFelix — APPROVED** [review]
/approve

**Pull request merged**